internal/snap_printer/snap_printer_electron_link_test.go

```diff
@@ -41,8 +41,7 @@ function main () {
   return a + b
 }
     `, `
-let __get_a__;
-let a, b;
+let __get_a__;let a, b;
```

Comment:
looks like this could have been left

Reply:
We talked about this offline, but basically in order to fix sourcemaps for now, I had to mess with the formatting of the generated code. The generated code is functional and properly sourcemapped but just doesn't look 100% formatted correctly. I logged this to address this later: https://github.com/cypress-io/esbuild/issues/11

Reply:
Added comment for this here: [`f972bb9` (#10)](https://github.com/cypress-io/esbuild/pull/10/commits/f972bb944f1be3e859325b53b7fee7b275ec5097)